### ISSUE TITLE
Cannot change the realm of a poller/scheduler

### ISSUE THREAD

**Issue by G-GU** [naparuba/shinken#2001]
Hello,

I have a Shinken configuration with 
 - one server for arbiter, reactioner, receiver, broker and IHM
 - 2 servers with one scheduler and one poller on each.

The 2 scheduler/poller are on the same realm.

I want to create a new realm and have one scheduler/poller on each realm.

I updated the file realm.cfg with the new realm.

I stopped all services, deleted all retention files

I changed the realm in the /appli/shinken/etc/myCompanyName/daemons/daemons_adm.cfg (all the other config files are generic with "locolhost") as below for scheduler and poller. I did this change on arbiter (and on poller.scheduler event if it's useless).

	define scheduler {
		scheduler_name  scheduler_XXX
		address         10.xx.xx.xx
		port            7768
		spare           0

		modules         scheduler_retention, bridge_XXX

		#realm           realm_OLD
		realm           realm_NEW
	
	}
	define poller {
		poller_name     poller_XXX
		address         10.xx.xx.xx
		port            7771
		spare           0

		min_workers     1
		max_workers     50

		use_ssl         0
		hard_ssl_name_check   0

		#realm           realm_OLD
		realm           realm_NEW
	}

But the arbiter don't start. 

With "journalctl -xe" I see below info. 

shinken-arbiter[28483]: [1589908960] INFO: [Shinken] realm_NEW: (in/potential) (schedulers:1) (pollers:1/1) (reactionners:0/0) (brokers:1/1) (receivers:1/1)

 [...]
 
shinken-arbiter[28483]: [1589908960] INFO: [Shinken] Things look okay - No serious problems were detected during the pre-flight check
shinken-arbiter[27242]: [1589919541] INFO: [Shinken] [Arbiter] Serializing the configurations...
shinken-arbiter[27242]: [1589919541] INFO: [Shinken] Using the multiprocessing serialization pass
shinken-arbiter[28483]: [1589908960] ERROR: [Shinken] Something goes wrong in the configuration serializations, please restart Shinken Arbiter
 
systemd[1]: shinken-arbiter.service: control process exited, code=exited status=2

This action is done on my Dev environment. The production environment has 2 realms and work fine with identical config files (except the IP, server names ...).

I suppose that I missed one parameter, but where ? 

Can anyone help ?

**Comment by geektophe** [comment]
I'm afraid I arrive a little bit late... do you stall face the same issue ?

**Comment by G-GU** [comment]
Thanks for answering. Issue is solved. I did not closed te post before because the forum was looking like dead.

I had few issue. I suppose that this one was due to comment I put after a parameter in the config file. I put a "#" (like for a line comment) instead of a ";" (for a comment at the end of a line).
The scheduler don't find any error when reading the config file but send the parameter value and the comment to the poller. I had to put debug in the Python code on the poller to see the issue.
